views/showcase.html

```diff
@@ -97,7 +97,7 @@ <h3 class="text-xl font-semibold">
             </dl>
           </a>
           {% endfor %}
-          {% for view in resource.views %}
+          {% for view in dataset.resources.views %}
```

Comment:
This change looks like not related to this commit message.

Reply:
I made error while I'm trying to see impact of using `dataset` instead of `resources`. I fixed this error.